### DISCUSSION TITLE
Leave it as MKV instead of MP4

### DISCUSSION THREAD

**Discussion by mrmuiz** [RandomNinjaAtk/arr-scripts#338] [Q&A]
How can i run this script but dont make it into a  MP4?

**Comment by RandomNinjaAtk** [comment]
By default it should be MKV only....

**Reply by mrmuiz** [reply]
output-format = mp4
output-extension = mp4

should be set to :

output-format = mkv
output-extension = mkv  ?

**Reply by RandomNinjaAtk** [reply]
What file are you referencing... ?

If your referencing SMA.ini, you must have changed it because the default provided is mkv....

**Reply by mrmuiz** [reply]
Yes. But its a long time ago. I needed mp4 for a LG tv for display Dolbyvision. But my 2nd Radarr and Sonarr are for HD. So it doesnt need to be MP4. I want MKV on those 2. Thats why i asked.
